File: Arreglos/Java/Filas_DI_S.java
public class Filas_DI_S{
    public static void main(String[] ars){

        int DIMENSION = 5;
        int Matriz[][] = new int[DIMENSION][DIMENSION];

        for(int Filas = 0; Filas < DIMENSION; ++Filas){
            for(int Columnas = 0; Columnas < DIMENSION; ++Columnas){
                
                if(Columnas < Filas){
                    System.out.print("    ");
                }else{
                System.out.print(Filas+","+Columnas+" ");
                }
            }

            System.out.println();

        }

    }
}

File: Arreglos/Python/Filas_DI_S.py
'''
Author: "Space Team"
Date: 12/02/2022
Description: Printing the upper diagonal of a matrix traversed by rows
'''

#Input
range = [0,1,2,3,4]
Dimension = 5
filas = 0
columnas = 0

#Algorithm
for filas in range:
    Dimension = Dimension-1    
    for columnas in range:
        if columnas <= Dimension:
            print(f"({filas},{columnas}) ", end= "")    
    print(" ")   

#Output        